vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -183,7 +184,7 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
         permissions: '0644'
         content: |
           {
-            \"insecure-registries\" : [\"192.168.111.1:5000\"]
+            \"insecure-registries\" : ["{{ EXTERNAL_SUBNET_V4_HOST }}":"{{ REGISTRY_PORT }}"]
```

Comment:
Maybe EXTERNAL_SUBNECT_V4_HOST is not the best variable to use there. you should probably use a variable that is IP version agnostic.

Reply:
Fixed thanks